packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -24,13 +25,13 @@ const TABS = {
     GLOSSARY: 'glossary',
 };
 
-const Info = ({ handleSelect, item, list }: TInfo) => {
+const Info = ({ handleSelect, item, is_multiplier_fx, list }: TInfo) => {
```

Comment:
There was 2 approaches for passing flag `is_multiplier_fx` in `<TradeCategories />`:
1) import and use `observer` and `useTradeStore` here (in _contract-type-info_ file)
2) pass it as a prop from parent component, which has already using `observer` and hooks.
I decided to stick to the 2d one.
I'm also thinking of renaming `is_multiplier_fx`  ---> `show_alternative_text`.  In the parent component it can be
`show_alternative_text = {is_vanilla_fx || !cancellation_range_list?.length}`. This might allow to use one flag for both `vanilla_fx` and `multiplier_fx` (and for future `fx` contracts). What do you think? Is it good idea or better to leave like this.